test/e2e/alertmanager_test.go

```diff
@@ -28,86 +28,83 @@ import (
 )
 
 func TestAlertmanagerCreateDeleteCluster(t *testing.T) {
+	ctx := testFramework.NewTestCtx(t)
+	defer ctx.CleanUp(t)
+	ctx.BasicSetup(t, framework.KubeClient)
+
 	name := "test"
 
 	defer func() {
```

Comment:
If we're testing the actual deletion here it makes more sense to actually put the delete after the create instead of deferring it.